### DISCUSSION TITLE
hold command help pls

### DISCUSSION THREAD

**Discussion by CODICMAN** [sezanzeb/input-remapper#313] [Q&A]
I want a macro so that whenever i keep pressing my mouse click it automatically clicks every 50 miliseconds 
I have tried using hold command but I am really bad with it so pls if anyone can help

**Comment by zealotous** [comment]
@CODICMAN Have you tried

```
k(BTN_LEFT).w(50)
```

More examples https://github.com/sezanzeb/input-remapper/blob/main/readme/examples.md

**Reply by DreadPirateLynx** [reply]
> @CODICMAN Have you tried
> 
> ```
> h(k(BTN_LEFT).w(50))
> ```
> 
> More examples https://github.com/sezanzeb/input-remapper/blob/main/readme/examples.md

Edited code for clarity's sake:
